docusaurus.config.js

```diff
@@ -74,8 +76,8 @@ const siteConfig = {
       copyright: `Copyright © ${new Date().getFullYear()} TradeTrust`,
     },
     algolia: {
-      appId: "BH4D9OD16A",
-      apiKey: "2c96d58935843a5e1a0f4e53ea18300e",
+      appId: "6DNGPTCB04",
+      apiKey: process.env.ALGOLIA_SEARCH_API_KEY || "dummykey",
```

Comment:
i dont think need `"dummyKey"` cos it will throw error when no apiKey is present.
 
Does it work with the `"dummyKey"`???

Also, there is a newer version of the docusaurus, wanna update it? v2.0.0-rc-1

Reply:
Also, wanna include a .env.example file? so that other developers can reference it to create their own .env file.

Reply:
no search won't work. exists only to not be a blocker in running the project locally. updated readme